persistence/schema/gov.go

```diff
@@ -1,442 +1,145 @@
-	ParamsTableSchema = `(
-		blocks_per_session INT NOT NULL,
-		app_minimum_stake TEXT NOT NULL,
-		app_max_chains SMALLINT  NOT NULL,
-		app_baseline_stake_rate	 INT NOT NULL,
-		app_staking_adjustment INT NOT NULL,
-		app_unstaking_blocks SMALLINT NOT NULL,
-		app_minimum_pause_blocks SMALLINT NOT NULL,
-		app_max_pause_blocks INT NOT NULL,
-
-		service_node_minimum_stake    TEXT NOT NULL,
-		service_node_max_chains       SMALLINT NOT NULL,
-		service_node_unstaking_blocks INT NOT NULL,
-		service_node_minimum_pause_blocks SMALLINT NOT NULL,
-		service_node_max_pause_blocks INT NOT NULL,
-		service_nodes_per_session SMALLINT NOT NULL,
-
-		fisherman_minimum_stake TEXT NOT NULL,
-		fisherman_max_chains SMALLINT NOT NULL,
-		fisherman_unstaking_blocks INT NOT NULL,
-		fisherman_minimum_pause_blocks SMALLINT NOT NULL,
-		fisherman_max_pause_blocks SMALLINT NOT NULL,
-
-		validator_minimum_stake TEXT NOT NULL,
-		validator_unstaking_blocks INT NOT NULL,
-		validator_minimum_pause_blocks SMALLINT NOT NULL,
-		validator_max_pause_blocks SMALLINT NOT NULL,
-		validator_maximum_missed_blocks SMALLINT NOT NULL,
-
-		validator_max_evidence_age_in_blocks SMALLINT NOT NULL,
-		proposer_percentage_of_fees SMALLINT NOT NULL,
-		missed_blocks_burn_percentage SMALLINT NOT NULL,
-		double_sign_burn_percentage SMALLINT NOT NULL,
-
-		message_double_sign_fee TEXT NOT NULL,
-		message_send_fee TEXT NOT NULL,
-		message_stake_fisherman_fee TEXT NOT NULL,
-		message_edit_stake_fisherman_fee TEXT NOT NULL,
-		message_unstake_fisherman_fee TEXT NOT NULL,
-		message_pause_fisherman_fee TEXT NOT NULL,
-		message_unpause_fisherman_fee TEXT NOT NULL,
-		message_fisherman_pause_service_node_fee TEXT NOT NULL,
-		message_test_score_fee TEXT NOT NULL,
-		message_prove_test_score_fee TEXT NOT NULL,
-
-		message_stake_app_fee TEXT NOT NULL,
-		message_edit_stake_app_fee TEXT NOT NULL,
-		message_unstake_app_fee TEXT NOT NULL,
-		message_pause_app_fee TEXT NOT NULL,
-		message_unpause_app_fee TEXT NOT NULL,
-		message_stake_validator_fee TEXT NOT NULL,
-		message_edit_stake_validator_fee TEXT NOT NULL,
-		message_unstake_validator_fee TEXT NOT NULL,
-		message_pause_validator_fee TEXT NOT NULL,
-		message_unpause_validator_fee TEXT NOT NULL,
-		message_stake_service_node_fee TEXT NOT NULL,
-		message_edit_stake_service_node_fee TEXT NOT NULL,
-		message_unstake_service_node_fee TEXT NOT NULL,
-		message_pause_service_node_fee TEXT NOT NULL,
-		message_unpause_service_node_fee TEXT NOT NULL,
-		message_change_parameter_fee TEXT NOT NULL,
-
-		acl_owner TEXT NOT NULL,
-		blocks_per_session_owner TEXT NOT NULL,
-		app_minimum_stake_owner TEXT NOT NULL,
-		app_max_chains_owner TEXT NOT NULL,
-		app_baseline_stake_rate_owner TEXT NOT NULL,
-		app_staking_adjustment_owner TEXT NOT NULL,
-		app_unstaking_blocks_owner TEXT NOT NULL,
-		app_minimum_pause_blocks_owner TEXT NOT NULL,
-		app_max_paused_blocks_owner TEXT NOT NULL,
+func init() {
+	govParamMetadataMap = parseGovProto()
```

Comment:
Here I am parsing the proto file since it's not going to change at runtime.

Reply:
ACK. Please add it as a comment